packages/plugin-ext-vscode/src/node/plugin-vscode-cli-contribution.ts

```diff
@@ -0,0 +1,54 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable } from 'inversify';
+import { Argv, Arguments } from 'yargs';
+import { CliContribution } from '@theia/core/lib/node/cli';
+import { PluginHostEnvironmentVariable } from '@theia/plugin-ext/lib/common';
+
+/**
+ * CLI Contribution allowing to override the VS Code API version which is returned by `vscode.version` API call.
+ */
+@injectable()
+export class PluginVsCodeCliContribution implements CliContribution, PluginHostEnvironmentVariable {
+
+    static VSCODE_API_VERSION = 'vscode-api-version';
+
+    protected vsCodeApiVersion: string | undefined;
+
+    configure(conf: Argv): void {
+        conf.option(PluginVsCodeCliContribution.VSCODE_API_VERSION, {
+            // tslint:disable-next-line:max-line-length
+            description: `Provides further refinement for the version used in VSCode API. Example: --${PluginVsCodeCliContribution.VSCODE_API_VERSION}=1.33.0`,
+            type: 'string',
+            nargs: 1
```

Comment:
How about moving the default value here? So one can also see it with `--help`.

Reply:
for me it's an external source, not the default one but I may be wrong ?

Reply:
agree it is nice that there is a fall back version inside the process

thought that for a user it would be nice to see the default version, let's see if someone asks for it, i can always check the code :)

Reply:
ok, I've included the default value in help